internal/goldendataset/testdata/pict_pairs_spans.txt

```diff
@@ -0,0 +1,289 @@
+Parent	Tracestate	Kind	Attributes	Events	Links	Status
```

Comment:
Is this is generated file? Can we have `generated` in the name and document how it is generated/can be regenerated?

Reply:
Changed file names to start with generated. I plan to do documentation after the entire solution is coded and I have chance to play around with the best approach for making changes and automating it all.